### PR TITLE
templates: CMS: updates to research guide

### PR DESCRIPTION
**Pull request by katilp**
Adresses part of the topics in https://github.com/cernopendata/opendata.cern.ch/issues/1518#issuecomment-346834340

### CODE REVIEW

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to learn about the terms under which I can access and use the CMS Open Data, and publish results obtained from them."
 
-- Go to ["Data preservation and open access policy"](https://cms-docdb.cern.ch/cgi-bin/PublicDocDB/ShowDocument?docid=6032) (if you are a CMS member see also "papers by CMS members using public data").
+- Go to ["Data preservation and open access policy"](http://opendata-dev.web.cern.ch/record/411) (if you are a CMS member see also ["Rules for use of open access CMS data by individual members of CMS"](https://cms-docdb.cern.ch/cgi-bin/DocDB/ShowDocument?docid=12242)).
```
Comment (atrisovic) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: You can exclude `http://opendata-dev.web.cern.ch` and just leave: `/record/411`, I think it should work at all places.

Comment (katilp) [reply cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: OK, good, I will change them on next round. There is quite a lot to do still

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 2]
```diff
@@ -24,27 +24,27 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
-- The 2011 data have more statistics, more diverse datasets, many associated MC sets, and a slightly more advanced VM environment. If you are immediately interested in maximal statistics and/or MC acceptance corrections you should go for 2011 data (more sophisticated).
+- The 2011/2012 data have more statistics, more diverse datasets, many associated MC sets, and a slightly more advanced VM environment. If you are immediately interested in maximal statistics and/or MC acceptance corrections you should go for 2011/2012 data (more sophisticated).
 - Information on the respective luminosities and pile-up rates vs time can be found on [this Twiki page](https://twiki.cern.ch/twiki/bin/view/CMSPublic/LumiPublicResults#Multi_year_Collisions_Plots).
-- If you want to try both datasets, you will need to use the appropriate VMs.
+- If you want to try both datasets, you will need to use [the appropriate VMs](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Remove `http://opendata-dev.web.cern.ch`.

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 3]
```diff
@@ -24,27 +24,27 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
 
 <center>–––</center>
 
-> "I want to install the CMS software environment on the virtual machine (separately for 2010 and 2011 data) which is needed for access to and analysis of CMS Research level data."
+> "I want to install the CMS software environment on the virtual machine (separately for 2010 and 2011/2012 data) which is needed for access to and analysis of CMS Research level data."
 
-- Go to ["Start analysing the data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-open-data), and
-- choose "2011" (default) or "2010".
+- Install [the approriate VM](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS), 2010 VM for 2010 data, and 2011 VM for 2011/2012 data.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Not strictly a review of this text, but should we re-name the second VM "2011/2012 VM"?

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 4]
```diff
@@ -24,27 +24,27 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
+- Go to "Getting started with CMS data:   ["Getting started with CMS 2010 data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-open-data) for 2010 or ["Getting started with CMS 2011 data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-2011-data) for 2011/2012 data FIXME check links
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: > Go to ["Getting started with CMS 2010 data"](/articles/getting-started-with-cms-open-data) for 2010 or ["Getting started with CMS 2011 data"](/articles/getting-started-with-cms-2011-data) for 2011/2012 data. [FIXME check links\

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 5]
```diff
@@ -24,27 +24,27 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
 
-**Note**: The 2010 (SL5) virtual machine will only work on 2010 data with CMSSW 4-2-8. The 2011 (SL6) virtual machine will only work on 2011 data and MC with CMSSW 5-3-32.
+**Note**: The 2010 (SL5) virtual machine will only work on 2010 data with CMSSW 4-2-8 (and other SLC5 based CMSSW releases). The 2011 (SL6) virtual machine will only work on 2011/2012 data and MC with CMSSW 5-3-32 (and other SLC6 based CMSSW releases).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: "SLC5-based" and "SLC6-based"

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 6]
```diff
@@ -59,30 +59,30 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
-- Go to 2010 CMS derived datasets,
+- Go to [2010 CMS derived datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&type=Dataset&subtype=Derived&experiment=CMS&year=2010&file_type=ig),
 - choose `Event display file derived from`… + the name of the CMS primary dataset you want (ZeroBias is known to be essentially empty).
 
 <center>–––</center>
 
 > "I want an alternative way to view these event displays."
 
-- Load the [CMS event display](http://opendata-dev.web.cern.ch/visualise/events/CMS),
+- Load the [CMS event display](http://opendata-dev.web.cern.ch/visualise/events/cms),
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Sorry I missed removing the first part of the URL earlier.

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 7]
```diff
@@ -92,22 +92,23 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to find out which 2011 data and MC sets exist, and how to get a feel for their content."
 
-- Go to 2011 CMS primary datasets or 2011 CMS simulated datasets,
-- choose a dataset, and read the comments.
+- Go to [2011 CMS primary datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&subtype=Collision&experiment=CMS&year=2011) or [2011 CMS simulated datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&subtype=Simulated&experiment=CMS&year=2011),
+- choose a dataset, and read the comments, read also about [simulated dataset names](http://opendata.cern.ch/about/CMS-Simulated-Dataset-Names) FIXME link
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `- choose a dataset, read the comments, and also read about [simulated dataset names](/about/CMS-Simulated-Dataset-Names) FIXME link`

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 8]
```diff
@@ -127,35 +128,34 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to find out how to use the trigger and trigger prescale information in the dataset I am interested in."
 
-- Go to CMS Trigger Information (currently for 2011 only).
-- This information is still very basic, and will be improved.
+- Go to [the Guide to CMS trigger system](http://opendata-dev.web.cern.ch/articles/guide-to-the-cms-trigger-system).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `Go to [the guide to the CMS trigger system]`

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 9]
```diff
@@ -127,35 +128,34 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
+- check the [list of validated runs](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=%22CMS%20list%20of%20validated%20runs%22).
 
 <center>–––</center>
 
 > "I want to find out whether I need condition data base information, and if so, how to access it."
 
 - Condition data are needed only on sophisticated examples using e.g. jet energy corrections (many of the simpler analysis/validation examples documented here do not use this).
 - Using condition data significantly slows down data access, so use them only if really needed. If so:
-    - go to CMS Condition Data for basic information (in contrast to what is stated there, they are NOT needed to run CERNVM in general, only to perform sophisticated tasks).
+    - see the instructions under to Option B in [Getting started with CMS 2011 data](CMS Condition Data for basic information (in contrast to what is stated there, they are NOT needed to run CERNVM in general, only to perform sophisticated tasks).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `- see the instructions below to Option B` or `- see the instructions to Option B`?

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 10]
```diff
@@ -127,35 +128,34 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
 > "I want to find the luminosity of my dataset, possibly constrained by using specific triggers."
 
-- Please check [CMS luminosity information](/record/1051),
+- Please check [CMS luminosity information](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=luminosity&type=Supplementaries&subtype=Luminosity),
 - decide on the triggers you want to use,
 - find out the runs/lumi sections in which these triggers were active and whether they were prescaled,
 - overlap with the available CMS Open Data samples (run range) and with the JSON data-quality selection.
     - If not prescaled, sum up the luminosity for the surviving runs/lumi sections.
-    - If prescaled, life is more complicated…
+    - If prescaled, life is more complicated, find the prescales as in [the trigger example code](FIXME)
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `- If prescaled, life is more complicated. Find the prescales as in [the trigger example code](FIXME)`

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 11]
```diff
@@ -178,17 +178,15 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
 
 - … for the MinimumBias, Commissioning, Mu or MuMonitor datasets:
-    - Go to 2010 Validation Utilities,
+    - Go to [Validation software](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&type=Software&subtype=Validation&experiment=CMS), FIXME Year in the search
     - choose and execute the corresponding Validation code.
 
 - … for the Multijet dataset (see also [here](http://hep.caltech.edu/cms/opendata/)):
-    - Go to 2010 CMS Tools,
-    - choose and execute `Razor filter and analyzer for SUSY searches`.
+    - choose and execute [Razor filter and analyzer for SUSY searches](http://opendata-dev.web.cern.ch/record/553).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `- choose and execute ["Razor filter and analyzer for SUSY searches"]`

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 12]
```diff
@@ -178,17 +178,15 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
 
 - … for the Electron (or Mu) dataset:
-    - Go to 2010 CMS Tools,
-    - choose and execute `Software to preprocess the CMS 2010 Muon and Electron datasets for the two-lepton/four-lepton analysis example of CMS open data`, then
-    - choose `Two-lepton/four-lepton analysis example of CMS 2010 open data` and compare PAT-tuples from the previous to those linked therein, or execute it on your new PAT tuples.
+    - choose and execute [Software to preprocess the CMS 2010 Muon and Electron datasets for the two-lepton/four-lepton analysis example of CMS open data](http://opendata-dev.web.cern.ch/record/200), then
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `- choose and execute ["Software to preprocess the CMS 2010 Muon and Electron datasets for the two-lepton/four-lepton analysis example of CMS open data"]`

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 13]
```diff
@@ -178,17 +178,15 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
+    - choose [Two-lepton/four-lepton analysis example of CMS 2010 open data](http://opendata-dev.web.cern.ch/record/101) and compare PAT-tuples from the previous to those linked therein, or execute it on your new PAT tuples.
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: `- choose ["Two-lepton/four-lepton analysis example of CMS 2010 open data"]`

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md (hunk 1 above)
Comment (katilp) [reply cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Done

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md (hunk 13 above)
Comment (katilp) [reply cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Thanks, all done 😃

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 14]
```diff
@@ -24,65 +24,65 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
+- If you want to try both datasets, you will need to use [the appropriate VMs](/search?page=1&size=20&tags=VM&experiment=CMS).
 
 <center>–––</center>
 
 > "I want to install the CMS software environment on the virtual machine (separately for 2010 and 2011/2012 data) which is needed for access to and analysis of CMS Research level data."
 
-- Install [the approriate VM](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS), 2010 VM for 2010 data, and 2011 VM for 2011/2012 data.
-- Go to "Getting started with CMS data:   ["Getting started with CMS 2010 data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-open-data) for 2010 or ["Getting started with CMS 2011 data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-2011-data) for 2011/2012 data FIXME check links
+- Install [the approriate VM](/search?page=1&size=20&tags=VM&experiment=CMS), 2010 VM for 2010 data, and 2011 VM for 2011/2012 data.
+- Go to: ["Getting started with CMS 2010 data"](/articles/getting-started-with-cms-2010-data) for 2010 or ["Getting started with CMS 2011 data"](/articles/getting-started-with-cms-2011-data) for 2011/2012 data FIXME check links
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: The `:` isn't necessary but it isn't a problem to be there.

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 15]
```diff
@@ -24,65 +24,65 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
...
 
 <center>–––</center>
 
 > "I want to produce some example physics distributions (e.g. inclusive di-muon spectrum analysis example directly from AOD or two-/four-lepton analysis example with intermediate ntuples)."
 
 - Install the CMS software as in the previous item (for 2010 or 2011).
-- Follow options A (di-muon spectrum, simpler) or B (two-/four-lepton example, more sophisticated). For 2011 option A, take the SingleMu or DoubleMu dataset index files from [http://opendata.cern.ch/record/32](http://opendata.cern.ch/record/32) or [http://opendata.cern.ch/record/17](http://opendata.cern.ch/record/17).
+- Follow options A (di-muon spectrum, simpler) or B (two-/four-lepton example, more sophisticated)in [Getting started with CMS 2010 data](/articles/getting-started-with-cms-2010-data). For 2011 option A, take the SingleMu or DoubleMu dataset index files from [http://opendata.cern.ch/record/32](/record/32) or [http://opendata.cern.ch/record/17](/record/17).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: - There's a space missing after the second set of brackets.
- `[Getting started with CMS 2010 data]` should probably be `["Getting started with CMS 2010 data"]`.

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md [hunk 16]
```diff
@@ -92,23 +92,23 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
 
 > "I want to find out which 2011 data and MC sets exist, and how to get a feel for their content."
 
-- Go to [2011 CMS primary datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&subtype=Collision&experiment=CMS&year=2011) or [2011 CMS simulated datasets](http://opendata-dev.web.cern.ch/search?page=1&size=20&q=&subtype=Simulated&experiment=CMS&year=2011),
-- choose a dataset, and read the comments, read also about [simulated dataset names](http://opendata.cern.ch/about/CMS-Simulated-Dataset-Names) FIXME link
+- Go to [2011 CMS primary datasets](/search?page=1&size=20&q=&subtype=Collision&experiment=CMS&year=2011) or [2011 CMS simulated datasets](/search?page=1&size=20&q=&subtype=Simulated&experiment=CMS&year=2011),
+- choose a dataset, read the comments, and read also about [simulated dataset names](/article/CMS-Simulated-Dataset-Names) FIXME link
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Should this be `/articles/`?

Comment (katilp) [reply cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md]: Thanks, all done!

**Comment by katilp** [comment]
Thanks, corrections done!

**Pull request merged**